new_app/old/js/transformedComponents/.module-cache/003a6f213a8aec8d9ed0c081a43165f7a6e5315d.js

```diff
@@ -0,0 +1,60 @@
+/**
+ * Created by omelvin on 5/21/15.
+ * @jsx React.DOM
+ */
+
+var RegistrationForm = React.createClass({displayName: "RegistrationForm",
+    getInitialState: function() {
+        return {email: this.props.studentID + "@illinois.edu", firstName: '', lastName: ''};
+    },
+    handleEmailChange: function(event) {
+        this.setState({email: event.target.value});
+    },
+    handleFirstNameChange: function(event) {
+        this.setState({firstName: event.target.value});
+    },
+    handleLastNameChange: function(event) {
+        this.setState({lastName: event.target.value});
+    },
+    handleSubmit: function(e) {
+        e.preventDefault();
+
```

Comment:
stylistic thing, but i think this looks better
```
if (text) {
 this.props.onCommentSubmit({parentID: parentID, commentText: text, video: videoName});
 this.setState({value: ''});
}
```

no need for return statements